Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -245,7 +245,14 @@ internal sealed class PythonLanguageClient : ILanguageClient, ILanguageClientCus
                         typeCheckingMode = typeCheckingMode,
                         useLibraryCodeForTypes = true,
                         completeFunctionParens = _advancedEditorOptions.CompleteFunctionParens,
-                        autoImportCompletions = _advancedEditorOptions.AutoImportCompletions
+                        autoImportCompletions = _advancedEditorOptions.AutoImportCompletions,
+                        indexing = _analysisOptions.Indexing,
+                        extraCommitChars = false,
+                        importFormat = _analysisOptions.ImportFormat,
+                        inlayHints = new LanguageServerSettings.PythonSettings.PythonAnalysisSettings.PythonAnalysisInlayHintsSettings {
```

Comment:
Booleans are false by default, so you it's redundant to initialize them here. But if you want to for clarity, that's fine.